yb-voyager/cmd/exportData.go

```diff
@@ -38,7 +38,7 @@ import (
 
 var exportDataCmd = &cobra.Command{
 	Use:   "data",
-	Short: "This command is used to export table's data from source database to *.sql files",
+	Short: "This command is used to export table's data from source database to *.sql files \nNote: For Oracle and MySQL, there is an experimental feature to speed up the data export, set the environment variable EXPERIMENTAL_FAST_EXPORT=1 for trying that out",
```

Comment:
Let's also point the user to the documentation? For now, it's not live, so we can just point the user to the documentation. 

How about:

`Note: For Oracle and MySQL, there is an experimental feature to speed up the data export, set the environment variable EXPERIMENTAL_FAST_EXPORT=1 to try it out. You can refer to Voyager Documentation for more details - https://docs.yugabyte.com/preview/migrate/`

Reply:
or directly to the export-data section, there will be a note in the Export data section itself for this right?
https://docs.yugabyte.com/preview/migrate/migrate-steps/#export-data

Reply:
@priyanshi-yb yup, that's better!